app/src/marketplace/modules/communityProduct/services.js

```diff
@@ -81,10 +81,10 @@ export const createJoinPartStream = async (productId: ?ProductId = undefined): P
 
     // Remove share permission to prevent deleting the stream
     try {
-        // $FlowFixMe
         const myPermissions = await getMyStreamPermissions(stream.id)
         const sharePermission = myPermissions.find((p) => p.operation === 'share')
         if (sharePermission) {
+            // $FlowFixMe is `sharePermission.id` a number or a string?
```

Comment:
Maybe should coerce it one way or the other

Reply:
Fixed that in bb5b70d00aa14bdc01a40fe1f8d712bdaa7d8e53.